platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/migratable/JsonUtils.java

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.admin.application.service.migratable;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.annotation.Nullable;
+import org.apache.karaf.features.Feature;
+import org.apache.karaf.features.FeaturesService;
+import org.codice.ddf.admin.application.service.Application;
+import org.codice.ddf.admin.application.service.ApplicationService;
+import org.codice.ddf.migration.MigrationException;
+import org.osgi.framework.Bundle;
+
+/**
+ * This class provides utility functions for dealing with Json objects returned by the <code>
+ * org.boon.json</code> library.
+ */
+public class JsonUtils {
+  private JsonUtils() {}
```

Comment:
Didn't want to mock those since thy were part of the implementation being tested anyway so static utility methods was good enough.

Redesigned to take advantage of Bean serialization/deserialization in Boon